src/config_brain_memory.js

```diff
@@ -454,3 +474,17 @@ brain.saveMemorySegments = function() {
     }
   }
 };
+
+brain.cleanAllMemory = function() {
+  const keys = _.map(_.keys(Memory), (key) => {
+    if (key !== 'players') {
+      delete Memory[key];
```

Comment:
a quick check `_.map(Memory, (value, key) => {return ex(key)}) === _.map(_.keys(Memory), (key) => {return ex(key)})`
and returns 
```
"players","rooms","segments","skippedRooms","username","myRooms","squads","orders","constructionSites","quests","creeps","needEnergyRooms","stats","powerBanks","layoutsCost","cpuStats","time","progress","summary"
```

Reply:
so i changed it to `_.map(Memory, (value, key)`

Reply:
That's not what I was talking about. your `map` call doesn't affect list of keys anyhow. Even if you are deleting all except of `'players'` key, your `const keys` array would contains `'players'`, and it would be reported to the console anyway